core/server/master/src/main/java/alluxio/master/file/FileSystemMasterOptions.java

```diff
@@ -12,199 +12,24 @@
 package alluxio.master.file;
 
 import alluxio.conf.ServerConfiguration;
-import alluxio.conf.PropertyKey;
-import alluxio.client.WriteType;
-import alluxio.grpc.CheckConsistencyPOptions;
 import alluxio.grpc.CompleteFilePOptions;
-import alluxio.grpc.CreateDirectoryPOptions;
-import alluxio.grpc.CreateFilePOptions;
-import alluxio.grpc.DeletePOptions;
-import alluxio.grpc.FileSystemMasterCommonPOptions;
-import alluxio.grpc.FreePOptions;
-import alluxio.grpc.GetStatusPOptions;
-import alluxio.grpc.GrpcUtils;
-import alluxio.grpc.ListStatusPOptions;
-import alluxio.grpc.LoadDescendantPType;
-import alluxio.grpc.LoadMetadataPOptions;
-import alluxio.grpc.MountPOptions;
-import alluxio.grpc.RenamePOptions;
-import alluxio.grpc.SetAclPOptions;
-import alluxio.grpc.SetAttributePOptions;
-import alluxio.grpc.TtlAction;
-import alluxio.security.authorization.Mode;
-import alluxio.util.ModeUtils;
-import alluxio.wire.LoadMetadataType;
+import alluxio.util.FileSystemOptions;
 
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
  * The file system class to set default options for master.
  */
 @ThreadSafe
-public final class FileSystemMasterOptions{
-
-  /**
-   * @return {@link FileSystemMasterCommonPOptions} with default values for master
-   */
-  private static FileSystemMasterCommonPOptions commonDefaults() {
-    return FileSystemMasterCommonPOptions.newBuilder()
-        .setTtl(ServerConfiguration.getLong(PropertyKey.USER_FILE_CREATE_TTL))
-        .setTtlAction(
-            ServerConfiguration.getEnum(PropertyKey.USER_FILE_CREATE_TTL_ACTION, TtlAction.class))
-        .setSyncIntervalMs(ServerConfiguration.getMs(PropertyKey.USER_FILE_METADATA_SYNC_INTERVAL))
-        .build();
-  }
+public final class FileSystemMasterOptions {
 
   /**
    * @return Master side defaults for {@link CompleteFilePOptions}
    */
   public static CompleteFilePOptions completeFileDefaults() {
```

Comment:
Is this the only method that keeps this file alive? Can't this be moved to FileSystemOptions?

Reply:
It could be, but I also wanted to separate on the master v. client-side options because there are many options objects but only a few are used on client. Even with one method I felt it warranted enough difference to keep this file